### PR TITLE
Catch pynvml.NVMLError_DriverNotLoaded Exception in gpu_stats

### PR DESCRIPTION
**Pull request by kilroythethird**
When trying to run under Manjaro i get:

```
Traceback (most recent call last):
  File "faceswap.py", line 5, in <module>
    import lib.cli as cli
  File "/run/media/nope/data/home/nope/workspace/fswap/faceswap/lib/cli.py", line 11, in <module>
    from lib.logger import crash_log, log_setup
  File "/run/media/nope/data/home/nope/workspace/fswap/faceswap/lib/logger.py", line 15, in <module>
    from lib.sysinfo import sysinfo
  File "/run/media/nope/data/home/nope/workspace/fswap/faceswap/lib/sysinfo.py", line 283, in <module>
    sysinfo = SysInfo()  # pylint: disable=invalid-name
  File "/run/media/nope/data/home/nope/workspace/fswap/faceswap/lib/sysinfo.py", line 19, in __init__
    gpu_stats = GPUStats(log=False)
  File "/run/media/nope/data/home/nope/workspace/fswap/faceswap/lib/gpu_stats.py", line 32, in __init__
    self.initialize()
  File "/run/media/nope/data/home/nope/workspace/fswap/faceswap/lib/gpu_stats.py", line 60, in initialize
    pynvml.nvmlInit()
  File "/usr/lib/python3.7/site-packages/pynvml.py", line 615, in nvmlInit
    _nvmlCheckReturn(ret)
  File "/usr/lib/python3.7/site-packages/pynvml.py", line 310, in _nvmlCheckReturn
    raise NVMLError(ret)
pynvml.NVMLError_DriverNotLoaded: Driver Not Loaded

```

Under Debian a non existent Nvidia card raised `NVMLError_LibraryNotFound` which is caught in `GPU_stats.initialize` and switched to CPU mode (A printed warning at that point would probably nice for misconfigured systems.)

On Manjaro this seem to raise a pynvml.NVMLError_DriverNotLoaded exception which this mini PR now also catches.

### CODE REVIEW

**Comment by kilroythethird** [comment]
I don't have an Nvidia card, so the exception is correct.
The reason i get the NVMLError_DriverNotLoaded and not NVMLError_LibraryNotFound on Manjaro might be the way manjaros [hardware detection](https://wiki.manjaro.org/index.php?title=Manjaro_Hardware_Detection_Overview) works.

Will update with the added log line later.

**Comment by kilroythethird** [comment]
Added the logging.

File: lib/gpu_stats.py [hunk 1]
```diff
@@ -58,7 +58,9 @@ def initialize(self):
                     if self.logger:
                         self.logger.debug("OS is not macOS. Using pynvml")
                     pynvml.nvmlInit()
-                except pynvml.NVMLError_LibraryNotFound:
+                except (pynvml.NVMLError_LibraryNotFound, pynvml.NVMLError_DriverNotLoaded):
+                    if self.logger:
```
Comment (torzdf) [inline lib/gpu_stats.py]: This needs to go to line 34 as otherwise macOS won't pick up the warning

Comment (kilroythethird) [reply lib/gpu_stats.py]: Right, makes sense. Changed

**Pull request merged**